### ISSUE TITLE
Definition of a group member can be strengthened

### ISSUE THREAD

**Issue by chelseakomlo** [mlswg/mls-architecture#71]
In the section "Groups, Members, and Clients", the definition of a group in MLS is the set of clients that have knowledge of the shared group key, which is established in the group key exchange. 

The spec then goes on to say that "Until a Member has contributed to the group secret, other members cannot assume she is a member of the group."

This phrasing of what constitutes a group member is quite weak and does not actually reflect the security properties of the underlying group key exchange. If a member has not contributed to the shared group secret, she cannot be considered a member of the group, as 1) she should not be able to learn the group key  (without acting adversarially outside of the protocol), and 2) other members cannot verify her membership

Instead, I would rephrase this description of a member/non-member to say something stronger, like "Until a Member has been added to the group by another member---thereby requiring that she contributes to the group secret in this process in such a way that is verifiable to other group members--she will not be able to participate as a member of the group."

**Comment by br-hale** [comment]
Good idea. An edit on this has been added to PR #82.

**Comment by kkohbrok** [comment]
I might be missing something here: Can't a party be added to a group (thus learning the group secret) and from that point never contribute to the shared group secret? That party would still be able to encrypt and decrypt application messages.

**Comment by beurdouche** [comment]
Hi @chelseakomlo could you please check the current document and propose a new phrasing if needed?
